Automattic/studio
# Add dropdown menu to clear Assistant conversation UI

Add dropdown menu to clear Assistant conversation UI 


https://github.com/Automattic/studio/assets/643285/9880362d-4557-436a-8a44-d9c52324ca24



Resolves Automattic/dotcom-forge#7528

## Proposed Changes

- Moves Clear Conversation button into DropdownMenu component
- Adds prompt to ask user to confirm clearing the conversation

## Testing Instructions

1. Navigate to Assistant tab
2. When a conversation has been started, click the Assistant Menu icon in the bottom right on the Assistant input
3. Click "Clear Conversation"
4. Confirm clearing the conversation using the window dialog
5. Observe the conversation has been cleared
 

## Pre-merge Checklist




## Todo
- [ ] Fix AIInput clear conversation test cases
- [ ] Close the dropdown menu when the conversation has been cleared
